### ISSUE TITLE
addition to "open branch" to include number of lines changed.

### ISSUE THREAD

**Issue by sofusalbertsen** [Praqma/git-metrics#5]
In addition to showing the amount of time, a given commit haven't been integrated, one could also measure how much (as in changed lines of code) change is laying on a given branch:

`git log --numstat --pretty="%H" commit1..commit2  | awk 'NF==3 {plus+=$1; minus+=$2} END {printf("+%d, -%d\n", plus, minus)}'`

**Comment by sofusalbertsen** [comment]
Snippet taken from:
https://stackoverflow.com/questions/2528111/how-can-i-calculate-the-number-of-lines-changed-between-two-commits-in-git

**Comment by sofusalbertsen** [comment]
I see this as an army knife of utills that can give you insights, where each metric should have a reason/description about _what_ this metric can tell you.
But maybe this is scope creep for your repo's intention?

**Comment by Neppord** [comment]
No, I don't think it's scope creep. I thinking it's a good idea and that we need to add it as new command or a flag on a the existing command.

This could also be a good oppertunity to make all the different endpoints "true" git commands by giving them the correct name. Like "git-branch-diffsize" or other?!
